### PR TITLE
mention new members who don't have a username and add open issues handler

### LINKED ISSUE #21: (feature) mention new members who don't have a username too
# Describe the feature you want to propose
When welcoming a new member, the bot should mention them in the welcome message, even if they don't have a username.
Right now it does mention their username but if they don't have one, it simply writes their first name:
https://github.com/JIITODC/bot/blob/bc475644f600d5c518bc1b35883f910d3f890fea/commands/welcome.py#L19-L22

# Implementation
You can use inline-style links to mention a person. For example, something like this will work as a mention of the `new_member`
```python
"[{first_name}](tg://user?id={new_member.id})" 
```

# Additional context
If its your first time contributing to the project, welcome :hugs: . Take a look at the [contributing guide](https://github.com/JIITODC/bot/blob/master/CONTRIBUTING.md) which is a step by step guide on getting the bot setup. 
We use the `python-telegram-bot` API wrapper. Here is a link to its documentation: https://python-telegram-bot.readthedocs.io/en/stable/index.html .
Do comment below something like 'I want to work on this issue' to let others know that you're working on this. Feel free to ask below for help with any doubts you have or problems you're facing. We're always glad to help. :v: :grinning:

### PR DESCRIPTION
**Pull request by Onyx2406**
## Description

updated welcome.py to mention new users who don't have username

updated help.py to add issues command

created issues.py

## Does this resolve any currently open issues?
Resolves #21 

## Screenshot/GIF showing resolved bug/enhancement (if applicable)
image

## Checklist
Just a friendly reminder to update dependencies/documentation if required. Do checkout if applicable.
- [ ] Updated the [requirements.txt](requirements.txt) (new dependency)
- [ ] Updated the [documentation](README.md) (setup instructions changed)

## Additional points worth mentioning
updated /bot/welcome.py to mention new members who don't have a username.
added issues.py in /commands.

### CODE REVIEW

File: commands/help.py [hunk 1]
```diff
@@ -21,7 +22,7 @@ def help(update, context):
     context.bot.send_message(
         chat_id=chat_id,
         text=help_text,
-        parse_mode=ParseMode.HTML,
+        parse_mode=ParseMode.MARKDOWN,
```
Comment (aayushmau5) [inline commands/help.py]: Any reason why do we need to change this back to `MARKDOWN`?

Comment (Onyx2406) [reply commands/help.py]: oh sorry I had just pasted my previous code which was before the last commit change today..

File: commands/help.py [hunk 2]
```diff
@@ -10,6 +10,7 @@ def help(update, context):
         "/xkcd@"+bot_name+" - Get an xkcd comic. Random if no argument given.\n" 
         "/links@"+bot_name+" - Get links to reach JODC.\n"
         "/meetup@"+bot_name+" - Get next info about next meetup.\n\n"
+        "/issues@"+bot_name+" - Find all the open issues here and contribute yourself!.\n"
```
Comment (aayushmau5) [inline commands/help.py]:
```suggestion
        "/issues@"+bot_name+" - Find all our open issues here.\n"
```

File: commands/issues.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        [
+            InlineKeyboardButton(
+                text="Open Issues", url="https://github.com/JIITODC/bot/issues?q=is%3Aopen+is%3Aissue"
+            ),
+        ],
+    ]
+    reply_markup = InlineKeyboardMarkup(buttons)
+    # update.message.reply_text(reply_markup=reply_markup)
+    context.bot.send_message(
+        chat_id=chat_id, text="Find all the issues here:", reply_markup=reply_markup
```
Comment (aayushmau5) [inline commands/issues.py]:
```suggestion
        chat_id=chat_id, text="Find all the open issues here:", reply_markup=reply_markup
```

**Comment by aayushmau5** [comment]
Just a little bit of change requested, otherwise everything looks alright to me.

**Review by aayushmau5 — APPROVED** [review]
LGTM. Thanks @Onyx2406 :rocket:

**Pull request merged**